ci/nightly/pipeline.template.yml

```diff
@@ -337,6 +337,28 @@ steps:
               composition: platform-checks
               args: [--scenario=RestartEntireMz, "--seed=$BUILDKITE_JOB_ID"]
 
+      - id: checks-backup-restore-before-manipulate
+        label: "Backup + restore before manipulate(#2)"
```

Comment:
Optional: Can we replace `(#2)` with something more expressive?

Reply:
The best I can do is say "between the two manipulate() phases" ?

Reply:
Ok, I think that gets too verbose